hikari/impl/entity_factory.py

```diff
@@ -3928,3 +3939,48 @@ def deserialize_sku(self, payload: data_binding.JSONObject) -> monetization_mode
             slug=payload["slug"],
             flags=monetization_models.SKUFlags(payload["flags"]),
         )
+
+    ###############
+    # POLL MODELS #
+    ###############
+
+    def _deserialize_poll_media(self, payload: data_binding.JSONObject) -> poll_models.PollMedia:
+        return poll_models.PollMedia(
+            text=payload["text"], emoji=self.deserialize_emoji(payload["emoji"]) if "emoji" in payload else None
+        )
+
+    def deserialize_poll(self, payload: data_binding.JSONObject) -> poll_models.Poll:
+        question = self._deserialize_poll_media(payload["question"])
+        expiry = time.iso8601_datetime_string_to_datetime(payload["expiry"])
+        allow_multiselect = payload["allow_multiselect"]
+        layout_type = poll_models.PollLayoutType(payload["layout_type"])
+
+        answers: typing.MutableSequence[poll_models.PollAnswer] = []
```

Comment:
```suggestion
        answers: list[poll_models.PollAnswer] = []
```

It's fine to use these in internal types (I don't even think we need the type tbh)